### ISSUE TITLE
for 'Write a Review' > REC reviews, change drop-down options

### ISSUE THREAD

**Issue by ipilipos** [cannalogue/CannaReviews#45]
in the write a review for REC,

change drop-down options for 'What was the reason for consumption",

to:
Relaxed
Happy
Uplifted
Energetic
Focused
Sleepy
Hungry
Pain-Free
Cheerful
Calm
Peaceful
Humorous
Excited
Blissful
Optimistic
Angry
Cranky
Gloomy
Restless

**Comment by dingxizheng** [comment]
@ipilipos Does it mean that for med and rec we'll a different list of reason for use

**Comment by ipilipos** [comment]
Yeah so medical stays the same as it is now, Rec is the one in this story for '
What was the reason for consumption?'
